comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -55,6 +56,8 @@ type rcClient struct {
 	// Tasks are separated from the other products, because they must be executed once
 	taskListeners     []types.RCAgentTaskListener
 	settingsComponent settings.Component
+	Log               log.Component
+	Config            coreConfig.Component
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Do you need those 2 components?

Reply:
yup! The StreamLogs command that i run require these 2 components:
https://github.com/DataDog/datadog-agent/blob/remote-config-stream-log/cmd/agent/subcommands/streamlogs/command.go#L182